exporter.go

```diff
@@ -276,7 +277,9 @@ func (e *Exporter) Cache(layersDir string, cacheStore Cache) error {
 }
 
 func (e *Exporter) tarLayer(layer identifiableLayer) (string, string, error) {
-	tarPath := filepath.Join(e.ArtifactsDir, launch.EscapeID(layer.Identifier())+".tar")
+	layerFilename := strings.ReplaceAll(layer.Identifier(), ":", "__")  // Replace colons, as they can be problematic for file names
```

Comment:
@ekcasey Here's the new spot where we tweak the tar file path instead of the underlying identifier.